raiden-ts/src/messages/types.ts

```diff
@@ -238,6 +239,29 @@ export const PFSCapacityUpdate = t.readonly(
 );
 export interface PFSCapacityUpdate extends t.TypeOf<typeof PFSCapacityUpdate> {}
 
+export const PFSFeeUpdate = t.readonly(
```

Comment:
This is the message we send to the PFS for the fees we are taking when mediating right?

Reply:
Yes, exactly, but even though we don't take fees yet, we need to send it at least once for each channel to tell PFS they can mediate through us and to calculate the fees we'll be taking as Zero.